docs/modules/operation/pages/topology/enlinkd/layer-2/bridge-discovery.adoc

```diff
@@ -1,240 +1,244 @@
-BridgeDomainDiscovery runs the specified algorithm over collected BFT and will produce a bridge domain or several bridge domains depending on the broadcast set of MAC addresses found.
-A bridge domains is a collection of shared segments based on common set of MAC addresses.
-
-BridgeDomainDiscovery does not support multi VLAN, the bridge network model identifies a bridge for every VLAN.
-Each VLAN has its own bridge forwarding table and its own spanning tree.
-To discover a bridge topology the algorithm has to be run against every bridge and every vlan.
-Actually the discovery is run only against the main VLAN.
-
-Bridge domains provide no information about layer 3; only a layer 2 map of the broadcast domains.
-You can determine which layer 3 interfaces map to layer 2 if your device supports the https://oidref.com/1.3.6.1.2.1.4.22[IpNetToMedia table].
-In this manner, we can associate a MAC address with the corresponding IP address and then the associated node.
-The Bridge Topology Updater combines the information stored in bridge domains with the ipnettomedia data and provided Bridge OnmsTopology.
-
-Whenever possible, the Bridge Topology Updater tries to associate a MAC address to an IP address and then to a node.
-It can happen that the specified MAC address and the IP address are not associated with a single node (for example, because there are duplicate nodes or because the nodes support a protocol like LACP).
-As the Bridge Topology Updater does not support LACP, and other similar aggregation protocols, we do not resolve the node and record the association found as a mac:ip relationship for a specific vertex.
-
-Transparent bridging is not loop free, so if you have loops you must enable the spanning tree protocol that detects loops and will put some ports in a blocking state to avoid loops.
-To get links it is necessary to perform some calculations that let us define the links.
-The SNMP agent must support the following MIBS to allow Transparent Bridge Discovery.
-
-.Supported MIBS from the Cisco-VTP MIB
-[options="header"]
-[cols="1,3,1"]
+Discovering xref:operation:topology/enlinkd/layer-2-discovery.adoc[layer 2 network links] using the bridge forwarding table (BFT) requires a special algorithm which is based on https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.383.8579&rep=rep1&type=pdf[_Topology Discovery for Large Ethernet Networks_].
+The gathered information is used to classify links as macLinks and bridgeLinks.
+
+== Definitions
+
+A macLink represents a link between a workstation or server that is identified by a MAC address.
+A bridgeLink represents a connection between backbone ports.
+
+A shared segment is a connection established among workstations or servers (made up of several MAC addresses) and backbone ports (for example, devices connected via a hub).
+A bridgeLink is a shared segment with only two bridge backbone ports.
+A macLink is a shared segment with only one bridge port and one MAC address.
```

Comment:
Swap the order to match the order these are mentioned in the previous paragraph.
```suggestion
A macLink is a shared segment with only one bridge port and one MAC address.
A bridgeLink is a shared segment with only two bridge backbone ports.
```